static/js/stream_create.js

```diff
@@ -173,22 +173,33 @@ function create_stream() {
     let is_web_public;
```

Comment:
Added lint changes in the next commit, so that the previous commit is easier to review.